static_src/components/form.jsx

```diff
@@ -0,0 +1,257 @@
+
```

Comment:
Yea I had similar thoughts. I think ideally, we'd be able to use a 3rd-party react forms library that allowed us to have control of the html for the forms. This would allow us to style our forms how we want while, ensure they're accessible, and use the logic of a maintained library. I think it would take a good amount of research to find the best 3rd party library for this.

I propose that the next time we have to create a form for the deck, if the form requires more functionality to be added to what we have here, then we'll commit to researching third party solutions rather then adding to this code. From there, we'll refactor any current form code to use the 3rd party library. If we really can't find a 3rd party library that meets our needs, we'll then commit to moving this form code into it's own 18f library and import it in.